packages/swap/src/routes/swap.ts

```diff
@@ -277,6 +280,14 @@ router.get(
       failedAt: failedSwap?.failedAt,
       failedBlockIndex: failedSwap?.failedBlockIndex ?? undefined,
       depositChannelAffiliateBrokers: affiliateBrokers,
+      depositChannelMaxBoostFeeBps: swapDepositChannel?.maxBoostFeeBps,
+      failedBoosts:
+        swapDepositChannel?.failedBoosts.map((failedBoost) => ({
+          failedAtTimestamp: failedBoost.failedAtTimestamp,
+          failedAtBlockIndex: failedBoost.failedAtBlockIndex,
+          amount: failedBoost.amount.toFixed(),
+        })) ?? [],
+      effectiveBoostFeeBps: swap?.effectiveBoostFeeBps,
```

Comment:
think i would not expose this for now if we dont need it

Reply:
We have it in the `fees` array, so it can be removed.